shared/constants/transaction.js

```diff
@@ -254,6 +254,8 @@ export const TRANSACTION_GROUP_CATEGORIES = {
  *  epoch time (ms).
  * @property {TransactionTypeString} type - The type of transaction this txMeta
  *  represents.
+ * @property {string} originalType - The current original type of the
```

Comment:
You should specify in which situations this will be present and what original type means,